src/lib/app/uuid.ts

```diff
@@ -50,6 +50,29 @@ export async function getAppVersionFromUuid(
   return appVersion.data;
 }
 
+/**
+ * Lookup an appInstallation by its UUID
+ *
+ * @param apiClient
+ * @param appInstallationId
+ */
+export async function getAppInstallationFromUuid(
+  apiClient: MittwaldAPIV2Client,
+  appInstallationId: string,
+): Promise<AppAppInstallation> {
+  if (!isUuid(appInstallationId)) {
+    throw new Error("Given UUID not valid.");
+  }
```

Comment:
tbh., I'd just rely on the API's response code here, instead of manually validating the input query. If it's not a valid installation ID, the API will tell you soon enough. 🤷

Reply:
My train of thought was that this function has no idea where the input comes from, it just gets passed a `string`.
If the given parameters came from an api-response or not should not be relevant for this function explicitly at the time of that check - or is that too petty?

Reply:
Oh, I've only just figured out how you meant that. Feed the string into the getAppinstallation request and see if the client/api can work with the string. Consider it done!